dora/core/client/fs/src/main/java/alluxio/client/file/PositionReadFileInStream.java

```diff
@@ -192,75 +198,87 @@ public int read(byte[] b, int off, int len) throws IOException {
 
   @Override
   public int read(ByteBuffer byteBuffer, int off, int len) throws IOException {
+    LOG.error("Reading len={} mPositionReader={}", len, mPositionReader);
```

Comment:
LOG.error is for error logging, may be debug?
and read is called pretty often, could we avoid logging in critical read/write path as much as possible (e.g. only log when have errors

Reply:
Sorry. This was added in coding phase and I did not change it back to 'debug'.

Reply:
Fixed.